lib/commands/smapi/customizations/smapi-hooks.js

```diff
@@ -7,13 +7,17 @@ const events = {
 
 const operationHooks = new Map();
 
-operationHooks.set('createSkillForVendorV1', new Map([[events.BEFORE_SEND, appendVendorId]]));
 operationHooks.set('createIspForVendorV1', new Map([[events.BEFORE_SEND, appendVendorId]]));
-operationHooks.set('getIspListForVendorV1', new Map([[events.BEFORE_SEND, appendVendorId]]));
+operationHooks.set('createSkillForVendorV1', new Map([[events.BEFORE_SEND, appendVendorId]]));
```

Comment:
I had add 3 more operations.We may want to add this hook automatically to any operation that has vendorId has non body parameter.